db/db_test.cc

```diff
@@ -2386,6 +2387,23 @@ TEST_F(DBTest, GetLiveBlobFiles) {
 
   ASSERT_FALSE(files.empty());
   ASSERT_EQ(files[0], BlobFileName("", blob_file_number));
+
+  ColumnFamilyMetaData cfmd;
+
+  db_->GetColumnFamilyMetaData(&cfmd);
+  ASSERT_EQ(cfmd.blob_files.size(), 1);
+  const BlobMetaData& bmd = cfmd.blob_files[0];
+  ASSERT_EQ(bmd.blob_file_number, blob_file_number);
+  ASSERT_EQ(bmd.blob_file_name, BlobFileName("", blob_file_number));
+  ASSERT_EQ(bmd.blob_file_size,
+            total_blob_bytes + BlobLogHeader::kSize + BlobLogFooter::kSize);
+
+  ASSERT_EQ(bmd.total_blob_count, total_blob_count);
+  ASSERT_EQ(bmd.total_blob_bytes, total_blob_bytes);
+  ASSERT_EQ(bmd.garbage_blob_count, 0);
+  ASSERT_EQ(bmd.garbage_blob_bytes, 0);
+  ASSERT_EQ(bmd.checksum_method, "CRC32");
+  ASSERT_EQ(bmd.checksum_value, "3d87ff57");
```

Comment:
Minor but you could use the constants defined above for expected values here as well, the same way you do for `total_blob_count` and `total_blob_bytes`.